frappe/public/js/frappe/views/kanban/kanban_board.js

```diff
@@ -640,12 +640,25 @@ frappe.provide("frappe.views");
 				title: frappe.utils.html2text(card.title),
 				disable_click: card._disable_click ? 'disable-click' : '',
 				creation: card.creation,
+				doc_content: get_doc_content(card),
 				image_url: cur_list.get_image_url(card),
 			};
 			self.$card = $(frappe.render_template('kanban_card', opts))
 				.appendTo(wrapper);
 		}
 
+		function get_doc_content(card) {
+			let fields = [];
+
+			JSON.parse(cur_list.board.fields || "[]").forEach(field => {
+				if (card.doc[field.fieldname] && card.doc[field.fieldname] !== card.title) {
+					fields.push(`<span>${__(field.label)}: ${__(card.doc[field.fieldname])}</span>`);
```

Comment:
@hrwX can we show the formatted value here?

Reply:
Also, why construct the HTML here? I'd suggest to move the HTML structure into `kanban_card.html`.